### PR TITLE
Feature/us318 help on search results

### PR DESCRIPTION
**Pull request by andymantell**

### CODE REVIEW

**Comment by woodingd** [comment]
:+1:

File: service/templates/search_results.html [hunk 1]
```diff
@@ -61,6 +61,14 @@ <h2 class="heading-small">
         <div id="pagination" style="text-align: center">Page {{ display_page_number }} of {{ results['number_pages'] }}</div>
       {% endif %}
     </div>
+    <div class="column-third">
+      <h2 style="border-top: 10px solid #005ea5;padding-top:25px;" class="heading-medium">Help finding the right property</h2>
```
Comment (RichardProsser) [inline service/templates/search_results.html]: h2 does not appear to be closed.

**Comment by RichardProsser** [comment]
Looks OK but I wonder if the 'style' elements should be more "CSS Classy".

**Comment by andymantell** [comment]
@RichardProsser This is what happens when we bring code in from the prototype verbatim and is something that we probably need a wider policy on. To what degree do we care at this point, given that the entire frontend is due a rebuild? It is not an accessibility or performance issue (It has zero impact on users) - the only issue with use of inline styles in frontend code is one of maintainability for us, which I think given the impending rebuild is potentially a non issue?

**Pull request merged**